bedrock/firefox/forms.py

```diff
@@ -10,8 +10,6 @@
 
 from lib.l10n_utils.dotlang import _lazy as _
 
-LANG_FILES = ['firefox/whatsnew-fx37']
-
```

Comment:
I believe this string as actually remnants of the even older /firefox/sms page. It may have only been en-US originally in this case, and just serves as a default message. We could remove it, but then perhaps it is better to leave something there as a default?